Yatima/Transpiler/Transpiler.lean

```diff
@@ -334,9 +327,9 @@ mutual
     let (argName, _) : Name × Expr := bindings.last!
     let argName ← mkName argName
     let recrArgs : List AST ← bindings.mapM fun (n, _) => mkName n
-    let mut [(n₁, rhs₁), (n₂, rhs₂)] ← rhs.mapM fun c => return (c.name, c.rhs)
+    let [(n₁, rhs₁), (_, rhs₂)] ← rhs.mapM fun c => return (c.name, c.rhs)
       | throw $ .custom "`mkListRecursor` must receive list constructors"
-    let (nilRHS, consRHS) :=
+    let (nilRHS, consRHS) := -- Arthur: why so specific?
```

Comment:
The bug that induced this code was from trying to compile List recursors which were external to other inductives, since if we replace the list representation, we have to maintain that everywhere. The `nil/cons` represent the two recursor cases for the list constructor.